### PR TITLE
Fix multiple accessibility errors

### PR DESCRIPTION
**Pull request by odero**
Fixes:
1. contrast error - use `#e50000` instead of `#ff0000`
2. required-children error - remove `ul[aria-labelledby]`, set `role="presentation"` in dropdown menu and `li` for proper `tablist` representation

### CODE REVIEW

**Review by marabooy — APPROVED** [review]
LGTM :shipit:

File: _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html [hunk 1]
```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
...
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>)/<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
```
Comment (gathogojr) [inline _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html:127]:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
```
This is a closing `</Apply> so we shouldn't have a space after `Apply`

File: _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html [hunk 2]
```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
...
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation </span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
```
Comment (gathogojr) [inline _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html:128]:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span><span style="font-family: Lucida Console"><span> </span></span></p>
```
This is a closing `</ValueAnnotation>` so we shouldn't have a space after `ValueAnnotation`

File: _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html [hunk 3]
```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
...
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">Apply </span><span style="color: #e50000">Function</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">RawValue</span>"</span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>          &lt;</span></span><span><span style="color: #a31515">Path</span><span style="color: #0000ff">&gt;</span>Version<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">Path</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
```
Comment (gathogojr) [inline _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html:135]:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
```
This is a closing `</Apply>` so we shouldn't have a space after `Apply`

File: _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html [hunk 4]
```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
...
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>"<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
```
Comment (gathogojr) [inline _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html:137]:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
```
This is a closing `</Apply>` so we shouldn't have a space after `Apply`

File: _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html [hunk 5]
```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
...
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation </span></span><span style="color: #0000ff">&gt;</span></span></p>
```
Comment (gathogojr) [inline _posts/2012-01-19-format-efficiency-take-2-really-clean-json.html:138]:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span></p>
```
This is a closing `</ValueAnnotation>`. We shouldn't have a space after `ValueAnnotation`

**Review by gathogojr — APPROVED** [review]
LGTM

**Pull request merged**